### PR TITLE
Minor API changes

### PR DESCRIPTION
**Pull request by justincase**
#95

### CODE REVIEW

File: lib/mollie/payment.rb [hunk 1]
```diff
@@ -23,6 +23,7 @@ class Payment < Base
                   :expires_at,
                   :failed_at,
                   :amount,
+                  :amount_captured,
                   :amount_refunded,
```
Comment (vernondegoede) [inline lib/mollie/payment.rb]: Can you also add the `authorizedAt ` parameter as well?

File: lib/mollie/customer/subscription.rb [hunk 2]
```diff
@@ -64,6 +66,14 @@ def times=(times)
         @times = times.to_i
       end
```
Comment (vernondegoede) [inline lib/mollie/customer/subscription.rb]: Can you add the `timesRemaining` parameter as well?

Comment (justincase) [reply lib/mollie/customer/subscription.rb]: 👍

File: lib/mollie/payment.rb (hunk 1 above)
Comment (justincase) [reply lib/mollie/payment.rb]: 👍

**Pull request merged**